test/Psr/CacheItemPool/CacheItemPoolDecoratorTest.php

```diff
@@ -9,39 +9,94 @@
 namespace LaminasTest\Cache\Psr\CacheItemPool;
 
 use Laminas\Cache\Exception;
+use Laminas\Cache\Psr\CacheItemPool\CacheException;
 use Laminas\Cache\Psr\CacheItemPool\CacheItemPoolDecorator;
-use Laminas\Cache\Storage\Adapter\AbstractAdapter;
+use Laminas\Cache\Psr\CacheItemPool\InvalidArgumentException;
+use Laminas\Cache\Storage\Adapter\AdapterOptions;
 use Laminas\Cache\Storage\Capabilities;
 use Laminas\Cache\Storage\StorageInterface;
+use Laminas\EventManager\EventManager;
+use LaminasTest\Cache\Psr\TestAsset\FlushableNamespaceStorageInterface;
+use PHPUnit\Framework\MockObject\MockObject;
 use PHPUnit\Framework\TestCase;
-use Prophecy\Argument;
 use Psr\Cache\CacheItemInterface;
 use stdClass;
 
-class CacheItemPoolDecoratorTest extends TestCase
+final class CacheItemPoolDecoratorTest extends TestCase
 {
-    use MockStorageTrait;
+    protected $defaultCapabilities = [
+        'staticTtl' => true,
+        'minTtl' => 1,
+        'supportedDatatypes' => [
+            'NULL'     => true,
```

Comment:
Why caps?

Reply:
This was just re-added. It was like this in the past and TBH, the only person who could answer that question is @marc-mabe as I did not fully understand any part of this package.

Reply:
This was just by the fact that `gettype(null)` returns it as upper case